diff --git a/ar6_ch6_rcmipfigs/notebooks/fig6_22_and_fig6_24/03-01_plot_fig6_22_dT_lineplot.py b/ar6_ch6_rcmipfigs/notebooks/fig6_22_and_fig6_24/03-01_plot_fig6_22_dT_lineplot.py
index 587db9a..0ca73c0 100644
--- a/ar6_ch6_rcmipfigs/notebooks/fig6_22_and_fig6_24/03-01_plot_fig6_22_dT_lineplot.py
+++ b/ar6_ch6_rcmipfigs/notebooks/fig6_22_and_fig6_24/03-01_plot_fig6_22_dT_lineplot.py
@@ -486,7 +486,7 @@ def get_difference_var(ds_DT, ref_year, var, last_y):
 # ## Main plot:
 
 # %%
-def make_main_plot(ds_rec, ref_year=ref_year, end_year=last_y, linewidth=2):
+def make_main_plot(ds_rec, ref_year=ref_year, end_year=last_y, linewidth=1):
     figsize = [7, 9]  # [6, 4]
 
     fig2, axs, ax_tot = get_fig_ax_tot(fgsize=figsize,
@@ -522,7 +522,7 @@ def make_main_plot(ds_rec, ref_year=ref_year, end_year=last_y, linewidth=2):
         else:
             ax.hlines(0, 1990, 2100, linewidth=.8, alpha=0.5)
 
-        add_uncertainty_bar(ax, var, linewidth=2.3, i_plus=1.7, s_y=ref_year, end_y=end_year)
+        add_uncertainty_bar(ax, var, linewidth=3.7, i_plus=3.6, s_y=ref_year, end_y=end_year)
         fix_ax(ax)
         ax.spines['bottom'].set_bounds(2019, 2100)
         bound_yax(ax)
@@ -678,7 +678,7 @@ for var in variables_erf_comp:
                ds_DT['ERF'].sel(variable=var, scenario=scn, year=slice(s_y, s_y)).squeeze()
         for _da, ax in zip([_da1, _da2], axs):
             _pl_da = _da  # .sel(percentile='median')#.mean(climatemodel)
-            _pl_da.plot(ax=ax, c=cdic[scn], label=scn, linestyle=lsdic[scn], linewidth=2)
+            _pl_da.plot(ax=ax, c=cdic[scn], label=scn, linestyle=lsdic[scn], linewidth=1)
     print(var)
 
     axs[0].set_title('GSAT change due to %s' % get_var_nicename(var))
diff --git a/env_rcmip_ch6.yml b/env_rcmip_ch6.yml
index 4616e7e..4f26162 100644
--- a/env_rcmip_ch6.yml
+++ b/env_rcmip_ch6.yml
@@ -6,6 +6,7 @@ channels:
 dependencies:
   - python=3.7
   - pip>19
+  - webcolors=1.13
   - ipywidgets
   - nb_black
   - iris
@@ -22,6 +23,7 @@ dependencies:
   - jupyter
   - ipykernel
   - nbconvert
+  - openpyxl=3.0.0
   - pip:
     - scons
     - black
